README.md

```diff
@@ -60,10 +60,20 @@ public interface ICacheAside
 The Add and remove methods are implemented with fire and forget, hence it does not need to be Async as this is handled by the StackExchange.Redis client.
 
 DoubleCache comes with the following implementations of this interface
-* LocalCache.MemCache - using System.Runtime.Memory
-* Redis.RedisCache - using StackExchange.Redis client
-* SubscribingCache - a decorator supporting push notifications of cache updates
-* PublishingCache - a decorator publishing cache changes
-* DoubleCache - a decorator wrapping a local and a remote cache
+* *[obsolete]* LocalCache.MemCache - using System.Runtime.Memory, does not support null items. 
+* LocalCache.WrappingMemoryCache - Allows storage of null items in Memory cache.* 
+* SystemWebCaching.HttpCache - An in memory cache using HttpContext.Cache.
+* Redis.RedisCache - using StackExchange.Redis client.
+* Redis.RedisStaleCache - Use redis as a stale cache in order to mitigate cache stampede. 
+* SubscribingCache - a decorator supporting push notifications of cache updates.
+* ExistingItemSubscribingCache - a decorator supporting push notifications, which will only update cache if the item already exists.
+* PublishingCache - a decorator publishing cache changes.
+* DoubleCache - a decorator wrapping a local and a remote cache.
```

Comment:
You might want to say what the default implementations `DoubleCache` uses when created from the factory.